Updates for new Managed Alerts for Magento Commerce alert policy (#7789)
magento/devdocs: src/cloud/project/new-relic.md

-  **Database query monitoring**—Locate and monitor database queries affecting performance.
-  **App Map**—View all application dependencies within your Magento site, extensions, and external services.
-  **Apdex scores**—Evaluate performance and create alerts that identify issues and notify you when they occur, such as site performance affected by a flash sale or web event. See [Apdex score].

> -  **Managed Alerts for Magento Commerce**–Use this New Relic alert policy to monitor application and infrastructure performance based on industry best practices. See [Monitor performance with the Magento alert policy](#monitor-performance-with-the-managed-alerts-for-magento-alert-policy).

Author: ```suggestion
-  **Managed Alerts for Magento Commerce**–Use this New Relic alert policy to monitor application and infrastructure performance based on industry best practices. See [Monitor performance with the Magento alert policy](#monitor-performance-with-managed-alerts).
```

For further information and examples, see [Introduction to New Relic Logs][New Relic Logs] and [Introduction to New Relic Insights][New Relic Insights].

> ### Monitor performance with the Managed Alerts for Magento alert policy

Reviewer: Whoa, I don't recommend a heading this long, because it looks bad in the page navigation.

```suggestion
### Monitor performance with Managed Alerts
```

### Monitor performance with alert policies

> Adobe provides the Managed alerts for {{ site.data.var.ee }} alert policy on Cloud Pro and Starter Production environments to track the following metrics:

Author: This statement is incorrect. Will update.

> Adobe provides the Managed alerts for {{ site.data.var.ee }} alert policy on Cloud Pro and Starter Production environments to track the following metrics:

Author: ```suggestion
Adobe provides the Managed alerts for {{ site.data.var.ee }} alert policy on Cloud Pro Production environments to track the following metrics:
```

Reviewer 2: Can I also suggest that it either be "Managed Alerts" or "managed alerts"? As the two words are connected it looks odd having one uppercase and one lowercase

-  [Apdex score]
-  error rate
-  disk space (available on Pro Production environments only)
-  CPU usage
-  memory usage

Based on industry best practices, this policy is a collection of alerts that set thresholds for warning and critical conditions that affect performance.

When your site experiences an infrastructure or application issue that triggers an alert threshold, New Relic sends alert notifications so that you can proactively address the issue.  See [Managed alerts for Magento Commerce][] in the _Magento Help Center_ for details about each alert type and troubleshooting steps to resolve the issues that triggered the alert.

> To use these policies, you must configure notification channels to receive the alert messages.

Author: yes-- this confusion is an artifact of previous implementation "Adobe-generated alert policies"

-  Verify that your Cloud environment is [connected to New Relic](#connect-to-new-relic)

{:.procedure}
To review the Managed Alerts for {{ site.data.var.ee }} policy:

1. Use your New Relic credentials to [log in to your New Relic account][New Relic login].

1. Locate the _Managed Alerts for Magento Commerce_ policy:

   -  In the top navigation menu, click **Alerts & AI** to open the _Applied Intelligence_ page.

   -  In the left navigation, click **Policies**.

>    -  At the top of the page, select the account and project from the dropdown menu if needed.

Reviewer: ```suggestion
   -  At the top of the page, select the account and project from the drop-down menu if needed.
```

To use the Managed Alerts for Magento Commerce policy to monitor your Production sites, you must configure at least one notification channel and map it to the alert policy.

Notifications about performance issues go to all channels associated with an alert policy when conditions on the application or infrastructure trigger an alert. You also receive notifications when an issue is acknowledged and closed.

New Relic provides templates for configuring different types of notification channels including email, Slack, PagerDuty, webhooks, and more. See the [Instructions for specific notification channels] in the New Relic documentation to review the prerequisites for using each type.

{:.procedure}
To configure a notification channel:

1. Choose a notification channel type, and complete any [prerequisite steps] required to connect the channel with the New Relic service.

1. Use your New Relic credentials to [log in to your New Relic account][New Relic login].

1. Create a new notification channel.

   -  In the top navigation menu, click **Alerts & AI** to open the _Applied Intelligence_ page.

   -  In the left navigation, click **Notification channels**.

> 

Reviewer: Is it important to comment about selecting an account or project from the drop-down list here like previously?

Author: I didn't think so, which is why I left it out.

-  Verify that your Cloud environment is [connected to New Relic](#connect-to-new-relic)

{:.procedure}
To review the Magento managed alert policies:

1. Use your New Relic credentials to [log in to your New Relic account][New Relic login].

1. Open your application.

   -  In the top navigation menu, click **APM** to open the application list.

   -  Click the application name to open the analytics dashboard.

1. In the top navigation menu, click **Alerts & AI** to open the _Applied Intelligence_ page.

> 1. In the Search Policies field, search for _Managed alerts_.

Reviewer: actually, the bold and italics here should be switched. The Search policies field is the location, and the search text is their action...hmmm

> 1. In the Search Policies field, search for _Managed alerts_.

Author: I took out this step because there's not a very long list of policies, and it's easy to spot the Magento one.

> 1. You should see the _Managed Alerts for Magento Commerce_ policy.

Reviewer: ```suggestion
1. In the _Policy_ list, you should see the **Managed Alerts for Magento Commerce** policy.
```

   ![Generated alert policies]({{ site.baseurl }}/common/images/cloud/cloud-new-relic-magento-managed-alerts.png){:width="650px"}

   {:.bs-callout-info}
   If you do not see this alert policy, submit a Magento support ticket. Include your project ID in the ticket. The policy is available on Pro and Starter Production environments only.

> 1. Click **Managed Alert policies for Magento Commerce** to review the alert conditions defined in the policy.

Reviewer: If it is the  "Managed Alerts policy" , then why is it the "Managed Alert policies" in the ui? I'm confused 😖 
```suggestion
1. Click **Managed Alerts for Magento Commerce** to review the alert conditions defined in the policy.
```

   ![Managed alerts list]({{ site.baseurl }}/common/images/cloud/cloud-newrelic-managed-alerts-for-magento.png){:width="650px"}

### Configure notification channels

To use the Managed alerts for Magento Commerce policy to monitor your Production sites, you must configure notification channels and map them to alert policies.

Notifications about performance issues go to all channels associated with an alert policy when conditions on the application or infrastructure trigger an alert. You also receive notifications when an issue is acknowledged and closed.

New Relic provides templates for configuring different types of notification channels including email, Slack, PagerDuty, webhooks, and more. See the [Instructions for specific notification channels] in the New Relic documentation to review the prerequisites for using each type.

{:.procedure}
To configure a notification channel:

1. Choose a notification channel type, and complete any [prerequisite steps] required to connect the channel with the New Relic service.

1. Use your New Relic credentials to [log in to your New Relic account][New Relic login].

> 1. Open your application.

Reviewer: Again, this step was not required for me. It may be like you said, because of our insider's viewpoint. But the Applied Intelligence page has a drop-down to select your project or account.
<img width="504" alt="Screen Shot 2020-09-01 at 11 11 19" src="https://user-images.githubusercontent.com/5817319/91877692-f5dbad00-ec43-11ea-8bbe-89694ecf4c20.png">
